thoth/package_extract/image.py

```diff
@@ -477,17 +489,14 @@ def _get_python_interpreters(path: str) -> dict:
             if len(parts) == 2 and parts[0] == "Python":
                 version_ = line.rstrip()
         except Exception as exc:
-            _LOGGER.warning(
-                "Failed to get python version: %s", str(exc))
+            pass
 
+        absolute_link = _get_absolute_link(path, py_path, 0)
         py_interpret = {
             "path": py_path[len(path):],
-            "link": None,
+            "link": absolute_link[len(path):] if absolute_link != py_path else None,
```

Comment:
`_get_absolute_link` can return `None` - a subscription using `[]` on `None` will be done here. Do something like above:

```python
        absolute_link = _get_absolute_link(path, py_path, 0)
        if absolute_link is not None:
            absolute_link = absolute_link[len(path):]
```

Reply:
Thanks, changed.